src/platform/site-wide/va-footer/components/CrisisPanel.jsx

```diff
@@ -73,7 +71,13 @@ export default function CrisisPanel() {
               <p>
                 Call TTY if you have hearing loss
                 <strong className="vads-u-margin-left--0p5">
-                  <Telephone contact={CONTACTS.SUICIDE_PREVENTION_LIFELINE} />
+                  <a
+                    className="no-wrap"
+                    href="tel:+18007994889"
+                    aria-label="8 0 0. 7 9 9. 4 8 8 9."
+                  >
+                    800-799-4889
+                  </a>
```

Comment:
Since this is loading as part of the footer (which is injected to TeamSite sites where web components are not supported yet), I'm replacing the deprecated React component with regular HTML markup.

I have verified it displaying in TeamSites locally:

![Screenshot 2023-09-13 at 2 23 23 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/788370c0-252b-4baa-831b-34894d5a7da6)